File: Programming-Basics/04. Console Input-Output/06. QuadraticEquation/QuadraticEquation.cs
using System;

class QuadraticEquation
{
    static void Main()
    {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());
        double xOne = 1d;
        double xTwo = 1d;
        double xTree = 1d;

        double d = Math.Pow(b, 2) - 4 * a * c;

        if (d < 0)
        {
            Console.WriteLine("No real roots!");
        }

        if (d == 0)
        {
            xOne = -b / (2 * a);
            Console.WriteLine(xOne);
        }

        if (d > 0)
        {
            xTwo = (-b + Math.Sqrt(d)) / (2 * a);
            xTree = (-b - Math.Sqrt(d)) / (2 * a);
            Console.WriteLine("x1 = {0} x2 = {1}" , xTwo, xTree);
        }
    }
}

File: JavaScript/02. Operators and Logic Flow/quadraticEquation.js
function solveQuadraticEquation(a, b, c) {
    let d = Math.pow(b, 2) - (4 * a * c);

    if(d > 0){
        let x1 = (-b + Math.sqrt(d)) / (2 * a);
        let x2 = (-b - Math.sqrt(d)) / (2 * a);

        console.log(+(x1 < x2 ? x1 : x2).toFixed(5));
        console.log(+(x1 > x2 ? x1 : x2).toFixed(5));
    }
    else if(d == 0){
        let x = -b / (2 * a);
        console.log(+x.toFixed(5));
    }
    else{
        console.log('No');
    }
}

//solveQuadraticEquation(1, -12, 36);